addons/max-pod/max-pod.addon

```diff
@@ -0,0 +1,8 @@
+# Name: max-pod
+# Description: Add-on to change the value of max-pods
```

Comment:
`Add-on to update the maximum pod value`

Reply:
@budhrg Thanks.. updating the description, WDYT about the name?

Reply:
Name is fine. :+1: